pkg/controller/monitor/monitor_controller.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021-2022 Tigera, Inc. All rights reserved.
+// Copyright (c) 2023 Tigera, Inc. All rights reserved.
```

Comment:
nit: should this be 2021-2023? I'm not so sure what we do with these.

Reply:
Done